pkg/logql/log/fmt.go

```diff
@@ -23,6 +24,26 @@ var (
 		"TrimPrefix": strings.TrimPrefix,
 		"TrimSuffix": strings.TrimSuffix,
 		"TrimSpace":  strings.TrimSpace,
+
+		// new function ported from https://github.com/Masterminds/sprig/
+		"lower":     strings.ToLower,
+		"upper":     strings.ToUpper,
+		"title":     strings.Title,
+		"trunc":     trunc,
+		"substr":    substring,
+		"contains":  func(substr string, str string) bool { return strings.Contains(str, substr) },
```

Comment:
It would be good to keep templates which have a similar signature as a downstream function and add a comment explaining why we need to reverse it, same as how it is done for `trimAll` below.